weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -1158,10 +1156,11 @@ MyApplet.prototype = {
     let counter = 0;       // Start counter, incremented for new days
 
     let forecast;
-    for (let i = 0; i < json.length; i++) {
-      let currentDate = new Date(json[i].dt * 1000);
+    weather.location.tzOffset = Math.round(json.city.coord.lon/15) * 3600;
```

Comment:
making sure we have the tz offset in the forecast, can't rely on the current weather call as both are async,